docs: documentent how to use kalix:run (#1578)
lightbend/kalix-jvm-sdk: maven-java/kalix-maven-archetype-event-sourced-entity/src/main/resources/archetype-resources/README.md

#[[
## Running Locally
]]#

> When running a Kalix application locally, at least two applications are required. The current Kalix application and its companion Kalix Proxy.

Reviewer: I find this phrase where we use the term "application" with different meanings a bit confusing.. 

```suggestion
When running a Kalix service locally, we need to have its companion Kalix Proxy running alongside.
```

Author: Totally agree.

> To start the applications locally, call the following command:

Reviewer: Being consistent from above, I would suggest...

```suggestion
To start your service locally, run:
```

```shell
mvn kalix:runAll
```

> This command will start your Kalix application and a Kalix Proxy using the included [docker-compose.yml](./docker-compose.yml) file.

Reviewer: ```suggestion
This command will start your Kalix service and a companion Kalix Proxy as configured in [docker-compose.yml](./docker-compose.yml) file.
```
